.github/workflows/main.yml

```diff
@@ -9,11 +9,13 @@ jobs:
     strategy:
       matrix:
         ruby:
+          - 3.0
```

Comment:
Should we aim for 3.1 as well, or would that be too much extra work?

Reply:
Will add and see

Reply:
added